### PR TITLE
revise MILModel docstring

### PR DESCRIPTION
**Pull request by wyli**
Signed-off-by: Wenqi Li <wenqil@nvidia.com>

### Description
improves the docstring:
Screenshot 2021-12-08 at 22 49 17

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: monai/networks/nets/milmodel.py [hunk 1]
```diff
@@ -11,23 +11,24 @@
...
-        mil_mode: MIL algorithm, available values:
-            "mean" - average features from all instances, equivalent to pure CNN (non MIL).
-            "max - retain only the instance with the max probability for loss calculation.
-            "att" - attention based MIL https://arxiv.org/abs/1802.04712.
-            "att_trans" - transformer MIL https://arxiv.org/abs/2111.01556.
-            "att_trans_pyramid" - transformer pyramid MIL https://arxiv.org/abs/2111.01556.
-            Defaults to ``att``.
+        mil_mode: MIL algorithm, available values (Defaults to ``"att"``):
+
+            - ``"mean"`` - average features from all instances, equivalent to pure CNN (non MIL).
```
Comment (Nic-Ma) [inline monai/networks/nets/milmodel.py:24]: May I know why add the "-" at the beginning?

Thanks.

Comment (wyli) [reply monai/networks/nets/milmodel.py:24]: Currently they are not displayed with bullet points without the "-"  https://docs.monai.io/en/latest/networks.html#milmodel

**Review by Nic-Ma — APPROVED** [review]

**Pull request merged**